Makefile

```diff
@@ -30,4 +30,20 @@ authors:
 	$Q cat AUTHORS GITAUTHORS  | sort -u > NEWAUTHORS
 	$Q mv NEWAUTHORS AUTHORS
 	$Q rm -f NEWAUTHORS
-	$Q rm -f GITAUTHORS
\ No newline at end of file
+	$Q rm -f GITAUTHORS
+
+testint:
+	@echo "Running Integration tests..."
+	$Q sudo GOPATH=$(GOPATH) go test -v -count=1 github.com/Azure/kubernetes-kms/tests/client
+
+test:
+	@echo "Running Unit Tests..."
+ifndef CI
+	@echo "Running Unit Tests outside CI..."
+	$Q go env
+	$Q dep ensure
+	go test -v -count=1 `go list ./... | grep -v client`
+else
+	@echo "Running Unit Tests inside CI..."
+	go test -v `go list ./... | grep -v client`
```

Comment:
From CircleCI logs, we seem to be getting this error `oauth.go:209] Couldn't open cloud provider configuration /tmp/kms_test196992206: &os.PathError{Op:"open", Path:"/tmp/kms_test196992206", Err:0x2}` how come the test still passes?

Reply:
nevermind I see what this is